### DISCUSSION TITLE
[Esame Ottobre 2023] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#375] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/393aa5f9-00d0-4fd6-b7c3-68ff5922346c)

**Comment by Frisbon** [comment]
Mi viene cosí
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/6846c16a-665d-4c72-aba3-138d0e7ffab3)
